config/routes.rb

```diff
@@ -10,7 +10,7 @@
   namespace :api do
     namespace :v1 do
       resources :merchants, only: [:index, :show]
-      resources :items, only: [:index, :show, :create, :update]
+      resources :items, only: [:index, :show, :create, :update, :destroy]
     end
```

Comment:
we can just use `resources :items` at this point (during the refactor)
And if we want to get fancy, we can change merchants to `except: :destroy`